cinema/serializers.py

```diff
@@ -51,7 +59,21 @@ class MovieDetailSerializer(MovieSerializer):
 
     class Meta:
         model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+        fields = (
+            "id",
+            "title",
+            "description",
+            "duration",
+            "genres",
+            "actors",
+            "image"
+        )
+
```

Comment:
Do you need class Meta here if it is inherited from MovieSerializer?

Reply:
Removed class Meta from here